README.md

```diff
@@ -51,7 +51,7 @@ With the Foundation libraries to encode/decode Json and Plist, one could ask: wh
 I have been working with many Mac admins recently, and many had to deal with Json, Plist and Xml data. While some were using a format-specific library like [jq](https://stedolan.github.io/jq/) to parse Json, others were using **awk**.  Each approach is valid, though it comes with some tradeoffs.
 
 #### Using a format-specific library
-You can use a library for each format. But I am not aware today of a library that unifies all of them. So, what you learnt with [jq](https://stedolan.github.io/jq/) cannot be reused to parse Plist data. You would have to learn to use **PlistBuddy** or the **defaults** command. With Scout, you can parse the same way Json, Plist and Xml data.
+You can use a library for each format. But I am not aware today of a library that unifies all of them. So, what you learned with [jq](https://stedolan.github.io/jq/) cannot be reused to parse Plist data. You would have to learn to use **PlistBuddy** or the **defaults** command. With Scout, you can parse the same way Json, Plist and Xml data.
```

Comment:
Just to make sure, to learn is not an irregular verb? I remember learning it at school like so 😄

Reply:
You're both right it seems:

https://conjugator.reverso.net/conjugation-english-verb-learn.html

Although I rarely see `learnt` it's still valid.